modules/lang-painless/build.gradle

```diff
@@ -28,6 +28,15 @@ integTestCluster {
   module project.project(':modules:mapper-extras')
 }
 
+Task additionalClusterTest = tasks.create(
+        name: "additionalClusterTest", type: RestIntegTestTask)
+additionalClusterTestCluster {
+  systemProperty 'es.script.exception_for_missing_value', 'true'
+  distribution = 'integ-test-zip'
+  module project // add the lang-painless module itself
+  module project.project(':modules:mapper-extras')
+}
```

Comment:
What tests is this actually running? It seems like this is running all the same rest tests. I would expect a test with in this PR which checks a deprecation message is emitted when the sysprop is false.